distribution/docs/src/main/resources/content/_managing/attributes-added-by-attribute-overrides-ingest.adoc

```diff
@@ -0,0 +1,15 @@
+:title: Attributes Added by Attribute Overrides (Ingest)
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How attribute overrides add attributes to metacards during ingest.
+:order: 04
+
+=== {title}
+
+<<_overriding_attributes,Attribute Overriding>> is the act of replacing existing attribute values with a new value.
+
+Attribute overrides can be configured for the <<_content_directory_monitor,Content Directory Monitor>>.
```

Comment:
❓ Can attribute overrides only be configured on ingest for CDM? If so, should we note that somehow in the flow diagram?

Reply:
Ya, they can only be configured for the CDM. However, I feel that the flow diagram's purpose is to show what steps happen in order. Adding more details than that would change the purpose (not necessarily a bad thing 🤷‍♂️).